tests/base/fixtures.py

```diff
@@ -0,0 +1,117 @@
+
+import pytest
+
+from django.db.models import Max
+
+from pontoon.base.models import (
+    Entity, ProjectLocale, Resource, Translation, User)
+
+
+@pytest.mark.django_db
+def test_fixtures_base0(entity0, resource0, project0, pl0,
+                        translation0, locale0):
+    assert entity0.resource == resource0
+    assert entity0.resource.project == project0
+    assert translation0.entity == entity0
+    assert pl0.project == project0
+    assert pl0.locale == locale0
```

Comment:
I think `pl` should be `project_locale`, especially if it's a fixture. It might be used in contexts where it will be much harder to guess what `pl` means.

Reply:
after typing `project_locale` a lot i changed it to this - my assumption is that people reading/writing tests in pontoon will know the convention

Reply:
I insist that it should be `project_locale`. We all use good IDEs with completion, and `pl0` reads a lot like `p10`, which makes it even more confusing. And I favor making it harder to write than read, because we read a lot more than we write.